hash/cassvisn_cart.xml

```diff
@@ -0,0 +1,193 @@
+<?xml version="1.0"?>
+<!DOCTYPE softwarelist SYSTEM "softwarelist.dtd">
+<!--
+license:CC0-1.0
+-->
+<!--
+The following games exist for this platform
+
+1 Yosaku
+2 Baseball
+3 Galaxian
+4 Big Sports 12
+5 Battle Vader
+6 PakPak Monster
+7 New Baseball
+8 Monster Mansion
+9 Astro Command
+10 Grand Champion*
+11 Monster Block
+12 Elevator Panic
+
+* = not dumped
+
+-->
+
+<!--
+Note on PRG size (from CPU documentation)
+
+Physically, 128 * 15 * 12 = 23,040 bits
+Logically, 127 * 15 * 12 = 22,860 bits (as final word of each page is inaccessible by the CPU)
+
+the ROM files below store 12 bit in a 16-bit word with 4 bits unused, so 128 * 15 * (12+4) 
+-->
+
+<softwarelist name="cassvisn_cart" description="Epoch Cassette Vision cartridges">
+
+	<software name="yosaku" supported="no">
+		<description>Yosaku</description>
+		<year>1981</year> <!-- 7/30/1981 -->
+		<publisher>Epoch</publisher>
+		<part name="cart" interface="cassvisn_cart">
+			<dataarea name="prg" size="0xf00">
+				<rom name="d777c005prog.s01" size="0xf00" crc="f917d2f0" sha1="b26baf2fd9b961b5a0e0475dfac93c18bd54cc4d" loadflag="load16_word_swap"/>
+			</dataarea>			
```

Comment:
The width and Endianness of these regions needs to match that of the CPU’s program space.  MAME only attempts to automatically work it out for a region with a tag matching a CPU device at the same level.  There’s no chance of working it out for a region that’s instantiated during media loading.

Reply:
this is a weird case of every word being 12-bit (in a 16-bit word in the ROM) rather than any real native endianness thing.  the hardware doesn't have a concept of bytes.  so a bit of a weird one anyway.  might need revisiting later.  Swapped in loading for now instead.